unfoldingWord/translationCore
# Migrate to manifest.json instead of tc-manifest.json

#### This pull request addresses:
Addresses issue 813. The beginning of our data migration from our tc-manifest to simply manifest.



#### How to test this pull request:

Load any project. If the project had a tc-manifest.json file it will now have a manifest.json file. If the project had no manifest it should now have a manifest.json file. If the project has a manifest.json file from a translation studio project the manifest.json file should now be updated to be compatable with translation core. If you open the manifest.json file you should see a new property of "tcInitialized" in the manifest to signify that this manifest has been initialized by translation core and is compatable with the latest version.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/915)
